ckanext/switzerland/helpers/date_helpers.py

```diff
@@ -210,3 +206,27 @@ def transform_date_for_solr(date):
         log.error("Exception {} occured on date transformation of {}"
                   .format(e, date))
         return None
+
+
+def get_localized_date(value):
+    """Take an isoformat date and return a localized date, e.g.
+    '24. Juni 2020'.
+    """
+    try:
+        dt = isodate.parse_datetime(value)
+        if isinstance(dt, datetime):
+            return localised_nice_date(dt, show_date=True, with_hours=False)
+    except Exception:
+        return None
+
+
+def get_datepicker_format(value):
```

Comment:
Just for consistency.. before you named it date_picker. Maybe you want to use datepicker all over, or date_picker? Wdyt?

Reply:
Thanks, I did want to use date_picker consistently but I missed a few datepickers. :)